components/SEO.tsx

```diff
@@ -1,185 +0,0 @@
-}
-
-const CommonSEO = ({ title, description, ogType, ogImage, twImage }: CommonSEOProps) => {
-  const router = useRouter()
-  return (
-    <Head>
-      <title>{title}</title>
-      <meta name="robots" content="follow, index" />
-      <meta name="description" content={description} />
-      <meta property="og:url" content={`${siteMetadata.siteUrl}${router.asPath}`} />
-      <meta property="og:type" content={ogType} />
-      <meta property="og:site_name" content={siteMetadata.title} />
-      <meta property="og:description" content={description} />
-      <meta property="og:title" content={title} />
-      {Array.isArray(ogImage) ? (
-        ogImage.map(({ url }) => <meta property="og:image" content={url} key={url} />)
-      ) : (
-        <meta property="og:image" content={ogImage} key={ogImage} />
-      )}
-      <meta name="twitter:card" content="summary_large_image" />
-      <meta name="twitter:site" content={siteMetadata.twitter} />
-      <meta name="twitter:title" content={title} />
-      <meta name="twitter:description" content={description} />
-      <meta name="twitter:image" content={twImage} />
-    </Head>
-  )
-}
-
-interface PageSEOProps {
-  title: string
-  description: string
-}
-
-export const PageSEO = ({ title, description }: PageSEOProps) => {
-  const ogImageUrl = siteMetadata.siteUrl + siteMetadata.socialBanner
-  const twImageUrl = siteMetadata.siteUrl + siteMetadata.socialBanner
-  return (
-    <CommonSEO
-      title={title}
-      description={description}
-      ogType="website"
-      ogImage={ogImageUrl}
-      twImage={twImageUrl}
-    />
-  )
-}
-
-export const TagSEO = ({ title, description }: PageSEOProps) => {
-  const ogImageUrl = siteMetadata.siteUrl + siteMetadata.socialBanner
-  const twImageUrl = siteMetadata.siteUrl + siteMetadata.socialBanner
-  const router = useRouter()
-  return (
-    <>
-      <CommonSEO
-        title={title}
-        description={description}
-        ogType="website"
-        ogImage={ogImageUrl}
-        twImage={twImageUrl}
-      />
-      <Head>
-        <link
-          rel="alternate"
-          type="application/rss+xml"
-          title={`${description} - RSS feed`}
-          href={`${siteMetadata.siteUrl}${router.asPath}/feed.xml`}
-        />
-      </Head>
-    </>
-  )
-}
-
-interface BlogSeoProps extends PostFrontMatter {
-  authorDetails?: AuthorFrontMatter[]
-  url: string
-}
-
-export const BlogSEO = ({
-  authorDetails,
-  title,
-  summary,
-  date,
-  lastmod,
-  url,
-  canonical,
-  images = [],
-}: BlogSeoProps) => {
-  const router = useRouter()
-  const publishedAt = new Date(date).toISOString()
-  const modifiedAt = new Date(lastmod || date).toISOString()
-  const authorNames = authorDetails?.map(({ name }) => name).join(', ')
-  const params = { title, desc: summary, authors: authorNames }
-  const query = new URLSearchParams(params).toString()
-  const defaultSocialBanner = `https://og-vercel-ruby.vercel.app/api/sekai?${query}`
```

Comment:
Are we dropping the social banners?

Reply:
This was unintentional, I'm sorry

Reply:
I have no clue how to reimplement this, the new `app/seo.tsx` uses the Next.js `Metadata` object which looks very different. Any guidance is appreciated.

Reply:
Figured it out (see [`5b62404` (#50)](https://github.com/project-sekai-ctf/sekai.team/pull/50/commits/5b624040d42e0e1b63835cf1860ecd9b26124746))